src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -179,11 +181,17 @@ function ReportActionCompose({
             return;
         }
         composerRef.current.focus(true);
+
+        // Restore the suggestions if onSelectionChange is not called on focused
+        if (isSelectionChangedOnFocus || !suggestionsRef.current) {
+            return;
+        }
+        suggestionsRef.current.restoreSuggestions();
```

Comment:
What is this for?

Reply:
`onSelectionChange` isn't called when composer gets focused(without any selection changes) on native platforms
In this case, we can restore the previous suggestion list. Previous suggestions are stored in `resetSuggestions` on blur handler

Reply:
Is this still nedeed?

Reply:
Without this, suggestions won't be shown unless selection is changed in native platforms. Refocusing or mouse clicking doesn't work.